system/services/presideObjects/PresideObjectService.cfc

```diff
@@ -1081,6 +1083,14 @@ component displayName="Preside Object Service" {
 		);
 
 		for (var record in records ) {
+			var formName = record.configurator__formname ?: "";
```

Comment:
You're right, it was just there to ensure unchecked checkboxes were recorded. Now implemented in the Javascript.